packages/airnode-node/src/api/unsafe-evaluate.ts

```diff
@@ -1,13 +1,117 @@
+import assert from 'assert';
+import async_hooks from 'async_hooks';
+import buffer from 'buffer';
+import child_process from 'child_process';
+import cluster from 'cluster';
+import console from 'console';
+import constants from 'constants';
+import crypto from 'crypto';
+import dgram from 'dgram';
+import dns from 'dns';
+import events from 'events';
+import fs from 'fs';
+import http from 'http';
+import http2 from 'http2';
+import https from 'https';
+import inspector from 'inspector';
+import module from 'module';
+import net from 'net';
+import os from 'os';
+import path from 'path';
+import perf_hooks from 'perf_hooks';
+import process from 'process';
+import readline from 'readline';
+import repl from 'repl';
+import stream from 'stream';
+import string_decoder from 'string_decoder';
+import timers from 'timers';
+import tls from 'tls';
+import trace_events from 'trace_events';
+import tty from 'tty';
+import url from 'url';
+import util from 'util';
+import v8 from 'v8';
+import vm from 'vm';
+import worker_threads from 'worker_threads';
+import zlib from 'zlib';
+
+const buildInNodeModules = {
+  assert,
+  async_hooks,
+  buffer,
+  child_process,
+  cluster,
+  console,
+  constants,
+  crypto,
+  dgram,
+  dns,
+  events,
+  fs,
+  http,
+  http2,
+  https,
+  inspector,
+  module,
+  net,
+  os,
+  path,
+  perf_hooks,
+  process,
+  readline,
+  repl,
+  stream,
+  string_decoder,
+  timers,
+  tls,
+  trace_events,
+  tty,
+  url,
+  util,
+  v8,
+  vm,
+  worker_threads,
+  zlib,
+};
+
 /**
  * This function is dangerous. Make sure to use it only with Trusted code.
  */
-export const unsafeEvaluate = (_input: any, code: string) => {
-  return eval(`
-    const ethers = require('ethers');
-    const input = _input;
+export const unsafeEvaluate = (input: any, code: string, timeout: number) => {
+  const vmContext = {
+    input,
```

Comment:
We stopped passing `ethers`, I think that one is rather necessary.

Reply:
Yeah, I think we should also pass all our airnode-* packages.

Reply:
I fear it will be confusing and harder to document and I do not see an use for using Airnode packages. If you this this is improtant we can keep it, but otherwise I'd just pass `ethers` and `axios`.

Reply:
It's difficult to know what's useful for the future.
At minimum anything can be achieved given the built-in node modules (like I re-implemented the `axios` requirement using built-in `https` in the target code snippet). Everything outside of the built in modules is convenience.
I'll remove the `airnode-*` require/imports :)